src/components/ClaimFilterWasCategorised.js

```diff
@@ -97,7 +109,7 @@ class ClaimFilterWasCategorised extends Component {
                                 onChange={e => this._onWasCategorizedCheckbox()}
                             />
                         }
-                        label="Categorized"
+                        label="Categorized by AI"
```

Comment:
The label should be added to the translations.

Reply:
Moved to translations